docs/source/index.rst

```diff
@@ -3,9 +3,10 @@
    You can adapt this file completely to your liking, but it should at least
    contain the root `toctree` directive.
 
-#########################
+####
 Home
-#########################
+####
+
 Vircadia is an open-source software where you can create and share virtual reality (VR) experiences. You can create and host your own VR world, explore other worlds, meet and connect with other users, attend or host live VR events and much more.
 
 The Athena metaverse provides built-in social features, including avatar interactions, spatialized audio and interactive physics. Additionally, you have the ability to import any 3D object into your virtual environment. No matter where you go in Vircadia, you will always be able to interact with your environment, engage with your friends, and listen to conversations just like you would in real life. 

```

Comment:
"The Athena metaverse..." ==> "The Vircadia metaverse..."

Also: This file is almost the same as home.rst. Perhaps only one is needed (e.g., the "HOME" sidebar link should navigate to index.html rather than home.html)?

Reply:
I'm not sure why they did it that way to be honest with you.

Reply:
Hmmm, maybe something to improve in the fulness of time.